packages/core/src/cart/line-item.ts

```diff
@@ -57,6 +57,7 @@ export interface LineItem {
     couponAmount: number;
     listPrice: number;
     salePrice: number;
+    retailPrice: number;
```

Comment:
@animesh1987,
I have updated the endpoint on backend side in a way that it should be always number. So that if retail price is not provided for a product, then 0 will be returned.
See
https://github.com/bigcommerce/bigcommerce/pull/52634/files#diff-b956c4c7fb4801b21e2e3c4cb2a725dfb819dd4b4c33a762270f13b4c3a6ec86R64
and
https://github.com/bigcommerce/bigcommerce/pull/52634/files#diff-16a680b18279d5d71064084b8613e1845784d3351afad2899c6a992dcc5ae7f1R69.

Also just updated the `Why` section. Should be clear enough now.